src/components/Modal/Modal.tsx

```diff
@@ -105,15 +111,16 @@ const Modal: ModalWithStaticComponents = forwardRef<HTMLDivElement, ModalProps>(
       ref={ref}
       {...restProps}
     >
-      <div className={styles.container}>
+      <Box className={styles.container}>
         <DialogContent
           aria-label={ariaLabel}
           aria-labelledby={ariaLabelledBy}
           className={contentClassnames}
+          style={{ ...maxWidth && { maxWidth } }}
```

Comment:
i would expect maxWidth to allow the same values as the Box. So it can be either tokens (sm, md, lg, etc...) or a css unit (10rem, 300px).